app/services/terms_of_use/decliner.rb

```diff
@@ -20,6 +23,10 @@ def perform!
 
     attr_reader :user_account, :version, :common_name
 
+    def validate_common_name
+      raise Exceptions::CommonNameNotFoundError if common_name.blank?
```

Comment:
Raise this with a message that we can render, that makes sense for a caller of the API. Probably something like 'Name for user must be present'